assets/css/style.css

```
@@ -321,8 +305,57 @@ a {
 .contact-bar-item svg path {
     fill: rgb(116, 135, 129);
 }
+
+
 @media ( max-width: 768px ) {
-    .contact-bar {
+    .navbar {
+        padding-right: 3px;
+        padding-left: 30px;
+    }
+
+    .navbar-menu {
         flex-direction: column;
     }
+
+    .navbar-btn {
+        margin-bottom:-12px
+    }
+
+    /* intro */
+    #intro-section {
+        padding: 150px 0;
+    }
+
+    /* about */
+    .skills {
+        flex-direction: column;
+        display: flex;
+    }
+
+    /* projects */
+    .projects-section .container {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .projects {
+        display: block;
+    }
+
+
+    /* contact-bar-item */
```

Nitpick: this should just be:

```suggestion
    /* contact */
```